substra/sdk/backends/remote/rest_client.py

```diff
@@ -212,7 +212,7 @@ def _add(self, name, exist_ok=False, **request_kwargs):
             if not exist_ok:
                 raise
 
-            key = e.pkhash or e.computePlanID
+            key = e.key or e.computePlanID
```

Comment:
actually e does not have a 'computePlanID' attribute, you can directly use `e.key`

Reply:
You're right (https://github.com/SubstraFoundation/substra/blob/bb0cf81b237d6e3f20a0dda034141193fa4db20d/substra/sdk/exceptions.py#L113-L117)

Reply:
@jmorel